warehouse/models/mart/gtfs_guidelines/_gtfs_guidelines.yml

```diff
@@ -46,3 +46,21 @@ models:
         description: |
           A string label for the GTFS "feature" associated with the given check. For example,
           "Compliant / On the Map".
+
+  - name: fact_daily_transitland_url_check
+    description: '{{ doc("transitland_url_check_table") }}'
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - dt
+            - url_type
+            - itp_id
+            - url_number
+    columns:
+      - name: dt
+      - name: url
+      - name: url_type
+      - name: itp_id
+      - name: url_number
+      - name: transitland.status
```

Comment:
I think these `transitland.<thing>` should be changed to `transitland_<thing>` based on how you renamed in the actual table?